CONTRIBUTING.md

```diff
@@ -15,56 +15,54 @@ that we follow.
 
 ## Reporting Issues
 
-Before reporting an issue, check our backlog of
-[open issues](https://github.com/containers/podman/issues)
-to see if someone else has already reported it. If so, feel free to add
-your scenario, or additional information, to the discussion. Or simply
-"subscribe" to it to be notified when it is updated.
-
-Note: Older closed issues/PRs are automatically locked, if you have a similar
-problem please open a new issue instead of commenting.
-
-If you find a new issue with the project we'd love to hear about it! The most
-important aspect of a bug report is that it includes enough information for
-us to reproduce it. To make this easier, there are three types of issue
-templates you can use, if you have a bug to report, please use *Bug Report*
-template or if you have an idea to propose, please use the *Feature Request*
-template or if your issue is something else, please use the default template.
-Please include as much detail as possible and try to remove the extra stuff
-that doesn't really relate to the issue itself. The easier it is for us to
-reproduce it, the faster it'll be fixed!
+Before reporting an issue, check our backlog of [open issues](https://github.com/containers/podman/issues) to see if someone else has already reported it.
+If so, feel free to add your scenario, or additional information, to the discussion.
+Or simply "subscribe" to it to be notified when it is updated.
```

Comment:
pre existing, but maybe add: Please do not add comment saying "+1" or similar without any new information. Instead use the thumbs up reaction on the original report/comment.

Thankfully these comments happen rarely enough so it is not a big problem but maybe nice to mention here.